apps/web/src/styles/theme.ts

```diff
@@ -117,13 +126,59 @@ const theme = extendTheme({
     },
     Button: {
       variants: {
-        solid: {
-          bg: "var(--chakra-colors-gray-100)",
+        solidPrimary: {
+          bg: light.blue,
+          color: "#FFF",
+          _hover: {
+            bg: light.blueDark,
+          },
+        },
+        solidSecondary: {
+          bg: light.grey[900],
+          color: "#FFF",
+          _hover: {
+            bg: light.grey[600],
+          },
+        },
+        outlinePrimary: {
+          border: "2px solid",
+          borderColor: light.blue,
+          color: light.blue,
+          _hover: {
+            borderColor: light.blueDark,
+            color: light.blueDark,
+          },
+          _dark: {
+            color: "#FFF",
+          },
+        },
+        iconButtonSolid: (props: StyleFunctionProps) => ({
```

Comment:
looks like this would've been simpler, no? right now we are mixing an outer `_dark` with the inner ones and it's hard to follow

```typescript
iconButtonSolid: {
  bg: light.grey[100],
  color: light.grey[900],
  _hover: { bg: light.grey[200] },
  _dark: {
    bg: dark.grey[100],
    color: dark.grey[900],
    _hover: { bg: dark.grey[200] },
  },
  }
}
```